db_postgresql/seeder.js

```diff
@@ -0,0 +1,68 @@
+const databaseConnection = require('./index.js');
+const generatedData = require('../server/DBgenerate.js');
+// const s3mock = require('../server/s3mock.js');
```

Comment:
You have a lot of commented code here. Please be sure to remove it for production.

Reply:
Deleted commented out code!